### PR TITLE
few basic changes

### PR DESCRIPTION
**Pull request by VikramTiwari**
restructure and a bit of md edits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,8 @@
...
 
-I also love lists, so I made a table of over 50 companies in edtech and social good I've found over the past few years!
+I also love lists, so I made a [table of over 50 companies in edtech and social good](organizations.md) I've found over the past few years!
 
-I also have a table of groups/meetups/events in education technology.
+I also have a [table of groups, meetups and events](events-and-meetups.md) in education technology.
 
-Please feel free to add more resources! I look forward to seeing your pull request :) -- and if you're unfamiliar with 
-git you are also free to email me at evy<dot>kassirer<at>gmail<dot>com
+Please feel free to add more resources! I look forward to seeing your pull request :) -- and if you're unfamiliar with git you [learn it here](https://try.github.io/) or feel free to email me at evy<dot>kassirer<at>gmail<dot>com
```
Comment (evykassirer) [inline README.md]: *can learn it here

File: organizations.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Welcome to my spreadsheet of edtech companies! 
+# Welcome to my spreadsheet of edtech companies!
```
Comment (evykassirer) [inline organizations.md]: haha thanks

Comment (VikramTiwari) [inline organizations.md]: haha renamed the file!

Comment (evykassirer) [inline organizations.md]: oh nice catch

**Pull request merged**